### ISSUE TITLE
Why the pml's output is sum of different verbalizations, is not the max of different verbalization ?

### ISSUE THREAD

**Issue by wa008** [timoschick/pet#63]
Thanks for sharing code!!!, I have a little question. 
>You can also define multiple verbalizations for a single label. For example, if you are unsure which words best represent the labels in a binary sentiment classification task, you could define your verbalizer as follows: `VERBALIZER = {"+1": ["great", "good", "wonderful", "perfect"], "-1": ["bad", "terrible", "horrible"]}`

If we want find the best represent verbalizations,  Why the pml's output is sum of different verbalizations, is not the max of different verbalization?
`cls_logits = cls_logits.sum(axis=1) / filler_len` in line 229 of [code](https://github.com/timoschick/pet/blob/a5a422e61aad569f3482f97c7720aba02b5f2ef2/pet/pvp.py)

Thank very much

**Comment by wa008** [comment]
Sorry, I think I know, because every verbalizations hava some probability to be outputed
